src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -97,7 +97,7 @@ class boss_nazan : public CreatureScript
             void IsSummonedBy(Unit* summoner) override
             {
                 if (summoner->GetEntry() == NPC_VAZRUDEN_HERALD)
-                    VazrudenGUID = summoner->GetGUID();
+					Vazruden = me->FindNearestCreature(NPC_VAZRUDEN, 5000);
```

Comment:
tabs2spaces plz :)

Reply:
I changed tabs to spaces,  Do I need another pull quesets?   Anyway  it's 1am,  I have to sleep.